model/project_parser.go

```diff
@@ -159,7 +158,37 @@ func (pp *ParserProject) Insert() error {
 }
 
 func (pp *ParserProject) MarshalBSON() ([]byte, error) {
-	return mgobson.Marshal(pp)
```

Comment:
I didn't mean to keep this MarshalBSON function updated, we don't use it anywhere and I tried seeing if we pass ParserProject anywhere since this fulfills the `bson.Marshaler` interface but I couldn't find any.

I kept it as-is and didn't repurpose just in case we use that interface anywhere.